Dockerfile

```diff
@@ -24,14 +24,14 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -y && \
 
 
 RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
-    wget --quiet https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
-    /bin/bash /Miniconda3-latest-Linux-x86_64.sh -b -p /opt/conda && \
-    rm Miniconda3-latest-Linux-x86_64.sh && \
-    /opt/conda/bin/conda install --yes conda
+    wget --quiet https://repo.continuum.io/miniconda/Miniconda3-4.0.5-Linux-x86_64.sh && \
```

Comment:
Cool. I think this version had some bug which had not been fixed yet. I will make the change and try it. I used 4.0.5 because I knew it worked.